msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -147,9 +228,15 @@ Anonymous users can view and interact with Adaptive Cards in the meeting chat. A
 
 * For an anonymous user, the user ID from `getContext` and the user ID received by the bot are different. It's not possible to correlate the two directly. If you need to track the user's identity between your tab and bot, you must prompt the user to authenticate with an external identity provider.
 
-* Anonymous users will see a generic app icon on bot messages and cards, instead of the app's actual icon. For example:
+* Anonymous users will see a generic app icon on bot messages and cards, instead of the app's actual icon.
+
+    :::image type="content" source="../assets/images/apps-in-meetings/app-icon.png" alt-text="Screenshot shows how the app icon displays for anonymous user.":::
+
+## Code sample
 
-    :::image type="content" source="../assets/images/apps-in-meetings/app-icon.png" alt-text="This screenshot shows you how the app icon displays for anonymous user.":::
+|Sample name | Description | .NET|
```

Comment:
Add one more column for `Manifest`

Link will be like `View`

URL will be like below:
https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/csharp/demo-manifest

Reply:
Updated